src/repository/repository.schema.js

```diff
@@ -0,0 +1,72 @@
+import { schema } from 'normalizr';
+import omit from 'lodash/omit';
+
+import { userSchema } from '../user/user.schema';
+
+/*
+export const userSchema = new schema.Entity('users', {}, {
+    idAttribute: user => user.login.toLowerCase(),
+
+});*/
+
+export const repoSchema = new schema.Entity(
+  'repos',
+  {
+    owner: userSchema,
+  },
+  {
+    idAttribute: repo => {
+      if (typeof repo.full_name === 'string') {
+        return repo.full_name.toLowerCase();
+      }
+
+      // In Events
+      return repo.name.toLowerCase();
+    },
+    processStrategy: entity => omit(entity, [
+      'url',
+      'html_url',
+      'url',
+      'forks_url',
+      'keys_url',
+      'collaborators_url',
+      'teams_url',
+      'hooks_url',
+      'issue_events_url',
+      'events_url',
+      'assignees_url',
+      'git_url',
+      'ssh_url',
+      'clone_url',
+      'svn_url',
+      'branches_url',
+      'tags_url',
+      'blobs_url',
+      'git_tags_url',
+      'git_refs_url',
+      'trees_url',
+      'statuses_url',
+      'languages_url',
+      'stargazers_url',
+      'contributors_url',
+      'subscribers_url',
+      'subscription_url',
+      'commits_url',
+      'git_commits_url',
+      'comments_url',
+      'issue_comment_url',
+      'contents_url',
+      'compare_url',
+      'merges_url',
+      'archive_url',
+      'downloads_url',
+      'issues_url',
+      'pulls_url',
+      'milestones_url',
+      'notifications_url',
+      'labels_url',
+      'releases_url',
+      'deployments_url',
+    ]),
+  }
```

Comment:
Now this 
![screen shot 2017-09-23 at 5 27 35 pm](https://user-images.githubusercontent.com/304450/30775314-379716b4-a091-11e7-9e99-5372f77283d7.png)
Looks definitely better than this:
![screen shot 2017-09-23 at 5 23 14 pm](https://user-images.githubusercontent.com/304450/30775317-45baddd4-a091-11e7-9665-6be3812e6fb9.png)